airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/BigQueryGcsDestinationAcceptanceTest.java

```diff
@@ -17,13 +17,12 @@
 import io.airbyte.integrations.standardtest.destination.DestinationAcceptanceTest;
 import java.nio.file.Path;
 import java.util.HashSet;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@Disabled
+// @Disabled
```

Comment:
reiterating from yesterday, we should re-disable these tests before merging. These tests are known to be broken https://github.com/airbytehq/airbyte/issues/29818

Reply:
Done